hsc/crawler.py

```diff
@@ -224,14 +235,14 @@ def get_submissions(self, submissions):
 
 def main():
 	offset = 0
-	limit  = 10 # you should change this
 
 	crawler = Crawler()
-
+	crawler.parse_script()
 	while(not crawler.authenticate()):
 		print('Auth was unsuccessful')
 
-	limit = input('Enter limit needed to crawl: ')
+	limit = crawler.options.limit or crawler.get_number_of_submissions()
```

Comment:
There's no need of requesting the number of submissions again, because we have stored the total number of submissions while authenticating the user. We can directly use `crawler.total_submissions`.